test/Dashboard.UnitTests/Indexers/FunctionIndexerTests.cs

```diff
@@ -78,5 +78,24 @@ public void CreateSnapshot_CreatesExpectedSnapshot()
             Assert.Equal(message.WebJobRunIdentifier.JobName, snapshot.WebJobName);
             Assert.Equal(message.WebJobRunIdentifier.RunId, snapshot.WebJobRunId);
         }
+
+        [Fact]
+        public void FunctionInstanceSnapshot_CompletelyRemovesLineBreaks()
+        {
+            FunctionInstanceSnapshot message = new FunctionInstanceSnapshot
+            {
+                Arguments = new Dictionary<string, FunctionInstanceArgument> 
+                {
+                    //{"message", new FunctionInstanceArgument(){ Value = "{\r\n  \"QRPoint\": {\r\n    \"X\": 0,\r\n    \"Y\": 0\r\n  },\r\n  \"TimesheetId\": 0,\r\n  \"HasReadableQrCode\": false,\r\n  \"HasSignature\": false,\r\n  \"BlobAddress\": \"https://myaccount.blob.core.windows.net/container/file.png\",\r\n  \"$AzureWebJobsParentId\": \"511f605d-14bf-46ca-b321-96b59d9e81d6\"\r\n}"} }
+                    {"message", new FunctionInstanceArgument(){ Value = "XXXXXX\r\nXXXXXXXXX\r\nYYYYYYY"} }
```

Comment:
If this special string was constructed with a specific length to trigger the bug, please add a comment to that effect